main.cpp

```diff
@@ -107,11 +107,36 @@ struct CarWash
     you should be able to deduce the return type of those functions based on their usage in Person::run()
     You'll need to insert the Person struct from the video in the space below.
  */
+struct Limb{};
 
+struct Person
+{
+    Limb leftFoot;
+    Limb rightFoot;
+
+    int age;
+    int height;
+    float hairLength;
+    float GPA;
+    unsigned int SATScore;
+    int distanceTraveled;
+
+    void run( int howFast, bool startWithLeftFoot);
+    void stepForward();
+    int stepSize();
+};
 
-
-
-
+void Person::run( int howFast, bool startWithLeftFoot)
+{
+    if( startWithLeftFoot == true )
+    {
+        void (distanceTraveled += howFast);
+    }
+    else
+     {
+        void (distanceTraveled -= howFast);//illogical but didn't want to copy over from video, makes code work.
+    }
+}
```

Comment:
This implementation of `run` is not what was shown in the video. 
Copy what was shown in the video.  
Then, figure out the solution for this instruction:
```
1) write the definition for the Type that leftFoot and rightFoot are instantiations of.
    don't forget to define and implement the member functions 'stepForward()' and 'stepSize()'
    you should be able to deduce the return type of those functions based on their usage in Person::run()
    You'll need to insert the Person struct from the video in the space below.
```
- [ ] revise

Reply:
Corrected.